gradle/libs.versions.toml

```diff
@@ -27,6 +26,7 @@ kotlin-test-annotations-common = { module = "org.jetbrains.kotlin:kotlin-test-an
 kotlin-gradle-plugin = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
 kotlinpoet = "com.squareup:kotlinpoet:1.18.1"
 kotlinx-coroutines = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core", version.ref = "coroutines" }
+kctfork = "dev.zacsweers.kctfork:core:0.5.1"
```

Comment:
can I suggest to follow the overall semantics? 
for example from this
``` kctfork = "dev.zacsweers.kctfork:core:0.5.1" ```
to this
``` kctfork = { module = "dev.zacsweers.kctfork:core", version = "0.5.1" } ```

Reply:
Why? It is just longer and we gain nothing. Unless we use `version.ref`, I'd keep the short version.

The only suggestion here is to use the lexicographic order to simplify the file reading.

Reply:
I've left the previous format as @DanySK suggested, for now. Also, I've adopted lexicographic order.